### PR TITLE
Add sampleStdev and sampleVariance methods. 

### PR DESCRIPTION
**Pull request by johnnaegle**
The stdev and variance functions, as exposed, continue to be population functions.

When the sample sizes are large enough, the values for sampleStdev and sampleVariance are very close. However, I was using a sample size of 6 and was seeing a large difference. Ultimately, I tracked this down to sample vs population standard deviations (and variances). This package uses stdev as a population functions where software such as postgres and applications such as google sheets use sample functions.

- Update the README with new functions
- Tests for sampleStdev and sampleVariance
- Expose 4 new functions: sampleStdev, populationStdev, sampleVariance, and populationVariance
- Alias stdev to populationStdev for compatibility
- Alias variance to populationVariance for compatibility
- Add test for populationStdev of only one thing

### CODE REVIEW

**Comment by johnnaegle** [comment]
I think I'd flip the names around: 

`stdevS`, `stdevP`, `varianceP`, `varianceS` instead of the longer names `sampleStdev`, `populationStdev`, `populationVariance`, `sampleVariance`.   

Thoughts?

**Comment by brycebaril** [comment]
I *think* I prefer the longer names, just for clarity. I've been thinking about other ways to expose this that might also make sense but I haven't come up with anything I like better yet.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stats-lite",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (brycebaril) [inline package.json]: We should probably bump minor instead of patch since this adds new functionality

Comment (johnnaegle) [reply package.json]: Bump to 2.2.0 and rebased

**Comment by brycebaril** [comment]
Published as v2.2.0
Thanks!

**Pull request merged**